Cargo.toml

```diff
@@ -23,3 +23,4 @@ async-channel = { version = "1" }
 once_cell = "1.17"
 prometheus = "0.13"
 argh = "0.1"
+axum = "0.6.12"
```

Comment:
Do we still need `hyper` as dependency?

Reply:
Yes, we still use lots of structs from `hyper`. `axum` is a thin wrapper.

BTW, `use crate` requires the crate to be listed in the `Cargo.toml` directly.

Reply:
Got it, thanks!